dataFiller/src/main/java/com/inctinctools/marathon2017/socmap/datafiller/model/ResponseServer.java

```diff
@@ -0,0 +1,20 @@
+package com.inctinctools.marathon2017.socmap.datafiller.model;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.List;
+
+public class ResponseServer {
```

Comment:
Bad naming - better to specify what API/service related to response:
if it's related to OSM - please use OSM in class naming:
eg - OSMResponse